QuickDeployment/QuickDeploymentEnvironment.md

```diff
@@ -1,52 +1,84 @@
+        - [1.4.1.3. How much time does this save?](#1413-how-much-time-does-this-save)
+        - [1.4.1.4. What if I have a larger environment? (> 1k nodes)](#1414-what-if-i-have-a-larger-environment--1k-nodes)
+        - [1.4.1.5. Can we brag about how fast we were able to get configured?](#1415-can-we-brag-about-how-fast-we-were-able-to-get-configured)
 
 <!-- /TOC -->
 
-## Summary
+## 1.2. QDE Components
 
-The quick deployment environment appliance provides an architecture that contains:
+The QDE appliance provides a unified architecture containing the following components:
 
-* Nexus (configured w/repositories)
-* Jenkins w/Internalizer Jobs
-* Chocolatey Central Management
-* Scripts for Internal Deployment
+* **Sonatype Nexus Repository OSS** (pre-configured with repositories) - This is your repository, where you'll be storing your Chocolatey packages (nupkg files).
+* **Jenkins** (w/Internalizer Jobs) - This is your automation engine, that lets you run tasks on-demand and on a schedule. It will host the PowerShell scripts to help you auto-internalize packages from the Chocolatey Community Repository
+* **Chocolatey Central Management** - This is your web dashboard for Chocolatey, that will allow you to track and monitor Chocolatey packages on your endpoint clients. You can see what packages are installed where, and whether or not they are out-of-date.
+* **Scripts for Internal Deployment** - Various scripts to help you confirgure this solution are included, for your convenience.
+
+> :warning: **WARNING**: This solution targets environments up to about **1000 nodes**.
+If you need a solution for a larger environment, QDE would still be suitable as a proof-of-concept; however, best practices would recommmend a distributed infrastructure, separating each component into its own discrete node.
+If you find yourself in need of a more scalable solution, please contact Support and we'll be more than happy to provide guidance for larger solutions.
+
+## 1.3. Links
+
+* [Quick Deployment Environment Setup](QuickDeploymentSetup.md)
+* Setting up proper DNS on QDE (DHCP by default; gonna wanna switch to static)
+* [QDE SSL Setup](QuickDeploymentSslSetup.md)
+* Setting up DNS on clients for accessing (make sure clients 
+* Installing the certificate on clients
+* Installing Chocolatey on clients (from QDE)
+* Opening up for remote access (for QDE)
 
+## 1.4. Related Articles
 
-*
+* [Organizational Deployment Guide](https://github.com/chocolatey/choco/wiki/How-To-Setup-Offline-Installation)
+* [[Client Install|Installation]]
+* [[Chocolatey Commercial Installation|Installation-Licensed]]
+* [[Automate Package Internalization|How-To-Setup-Internal-Package-Repository]]
 
-## Setup
+## 1.4.1. FAQs
 
-### VMDK
+### 1.4.1.1. What does the QDE (Quick Deployment Environment) include?
 
-Instructions here
+* Repository Server - Nexus (configured w/repositories)
+* Automation Pipelines - Jenkins w/Internalizer Jobs
```

Comment:
Replace 'w/Internalizer Jobs' with '(pre-configured with internalizer jobs)' for consistency.